Sources/VideoOptionsControlBar.swift

```diff
@@ -121,8 +128,6 @@ protocol VideoOptionsControlBarDelegate: class {
     
     override init(frame: CGRect) {
         repeatMode = .doNotRepeat
-        subtitleToggled = false
-        interfaceDisabled = false
```

Comment:
why was that removed as part of toggle fullscreen button image?

Reply:
Set their default values as false